src/api/store/action.py

```diff
@@ -34,7 +34,9 @@ def get_action_info(self, context: Context, args) -> Tuple[dict, MassEnergizeAPI
       return None, CustomMassenergizeError(e)
 
   def list_actions(self, context: Context, args) -> Tuple[list, MassEnergizeAPIError]:
-    try: 
+    try:
+      
+      filter_params = get_actions_filter_params(context.get_params())
```

Comment:
I don't think we want the filter_params for the actions.list route.  Only used in community portal